market/connector/kafka-connector/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+	"id": "kafka-connector",
+	"name": "Apache Kafka",
+	"description": "Produce and consume Apache Kafka messages.",
```

Comment:
@andreasbalsiger this new connector will be soon onboarded onto the market.
if you have any wished for the primary presentation here.
or the detailed product description https://github.com/axonivy-market/kafka-connector/blob/master/kafka-connector-product/README.md
... just let us know.